docs/pages/eas/json.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: eas.json
+description: Learn about available properties for EAS Build and EAS Submit to configure and override their default behavior from within your project.
+maxHeadingDepth: 4
+---
+
+import { EasJsonPropertiesTable } from '~/components/plugins/EasJsonPropertiesTable';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+import commonSchema from '~/public/static/schemas/unversioned/eas-json-build-common-schema.js';
+import androidSchema from '~/public/static/schemas/unversioned/eas-json-build-android-schema.js';
+import iosSchema from '~/public/static/schemas/unversioned/eas-json-build-ios-schema.js';
+import submitAndroidSchema from '~/public/static/schemas/unversioned/eas-json-submit-android-schema.js';
+import submitIosSchema from '~/public/static/schemas/unversioned/eas-json-submit-ios-schema.js';
+
+**eas.json** is the configuration file for EAS CLI and services. It is located at the root of a project next to the **package.json**. You can find the complete reference of all available schema properties for [EAS Build](/build/introduction) (`build`) and [EAS Submit](/submit/introduction) (`submit`) on this page.
```

Comment:
```suggestion
**eas.json** is the configuration file for EAS CLI and services. It is located at the root of a project next to the **package.json**. You can find the complete reference of all available schema properties for [EAS Build](/build/introduction) and [EAS Submit](/submit/introduction) on this page.
```

mostly because the `)` wraps and looks odd. 
<img width="838" alt="image" src="https://github.com/expo/expo/assets/90494/24f46f7e-75a5-43a6-9a66-a51789db4784">